fuzzers/Neo.VM.Fuzzer/Utils/CorpusManager.cs

```diff
@@ -0,0 +1,199 @@
+        public CorpusManager(string outputDir, string? initialCorpusDir = null)
+        {
+            _outputDir = outputDir;
+            _corpusDir = Path.Combine(outputDir, "corpus");
+            _crashesDir = Path.Combine(outputDir, "crashes");
+            _dosVectorsDir = Path.Combine(outputDir, "dos-vectors");
+
+            // Create output directories if they don't exist
+            Directory.CreateDirectory(_outputDir);
+            Directory.CreateDirectory(_corpusDir);
+            Directory.CreateDirectory(_crashesDir);
+            Directory.CreateDirectory(_dosVectorsDir);
+        }
+
+        /// <summary>
+        /// Loads the initial corpus from the corpus directory
+        /// </summary>
+        public void LoadCorpus()
+        {
+            if (string.IsNullOrEmpty(_corpusDir) || !Directory.Exists(_corpusDir))
+            {
+                return;
+            }
+
+            foreach (var file in Directory.GetFiles(_corpusDir, "*.bin"))
+            {
+                try
+                {
+                    byte[] script = File.ReadAllBytes(file);
+                    _corpus.Add(script);
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"Error loading corpus file {file}: {ex.Message}");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets a random script from the corpus
+        /// </summary>
+        /// <returns>A random script from the corpus</returns>
+        public byte[] GetRandomScript()
+        {
+            if (_corpus.Count == 0)
+            {
+                throw new InvalidOperationException("Corpus is empty");
+            }
+
+            return _corpus[_random.Next(_corpus.Count)];
+        }
+
+        /// <summary>
+        /// Saves a script that caused a crash
+        /// </summary>
+        /// <param name="script">The script that caused the crash</param>
+        /// <param name="exceptionMessage">The exception message</param>
+        public void SaveCrash(byte[] script, string? exceptionMessage)
+        {
+            string crashDir = _crashesDir;
+            string filename = $"crash_{DateTime.Now:yyyyMMdd_HHmmss}_{ComputeHash(script)}.bin";
+            string path = Path.Combine(crashDir, filename);
+
+            File.WriteAllBytes(path, script);
+
+            // Save metadata
+            if (!string.IsNullOrEmpty(exceptionMessage))
+            {
+                string metadataPath = Path.ChangeExtension(path, ".txt");
+                File.WriteAllText(metadataPath, exceptionMessage);
+            }
+        }
+
+        /// <summary>
+        /// Saves an interesting script that found new coverage
+        /// </summary>
+        /// <param name="script">The interesting script</param>
+        public void SaveInteresting(byte[] script)
+        {
+            // Add to corpus
+            _corpus.Add(script);
```

Comment:
Why this is added to corpus and the others don't?

Reply:
Still working on this, marked it as draft,